builder/index.html

```diff
@@ -266,11 +264,11 @@ <h3 >{{getMethodName(selectedMethod)}}
                               <form class="form-inline" role="form">
                                 <div class="form-group" ng-show="checkMajorVersion()">
                                   <label for="os_class">Version (Major):</label>
-                                  <input type="text" ng-pattern="versionRegex" style="width:50px" class="form-control" ng-change="updateClassContext()" ng-model="selectedMethod.OS_class.majorVersion"  placeholder=""></input>
+                                  <input type="text" ng-pattern="versionRegex" style="width:50px" class="form-control" ng-change="updateClassContext()" ng-model="selectedMethod.OS_class.majorVersion"  placeholder="">
                                 </div>
                                 <div class="form-group" ng-show="checkMinorVersion()">
                                   <label for="os_class">Version (Minor):</label>
-                                  <input type="text"  style="width:50px" ng-pattern="versionRegex" class="form-control" ng-change="updateClassContext()" ng-disabled="selectedMethod.OS_class.majorVersion === undefined || selectedMethod.OS_class.majorVersion === '' " ng-model="selectedMethod.OS_class.minorVersion"  placeholder=""></input>
```

Comment:
Reference is: http://w3c.github.io/html-reference/syntax.html#void-elements
Note that it seems you can self-close these tags  http://stackoverflow.com/questions/13232121/closing-html-input-tag-issue